flybywiresim/a32nx
# refactor(ecam): convert elec page to react

## Summary of Changes
- Rewrite the ELEC ECAM page into React.
- Rename BAT 10 to BAT 1.
- Rename BAT 11 to BAT 2.

Fixes #4515 (hopefully 🤨)

## Additional context



Discord username (if different from GitHub): SjotgunSjonnie#9623

## Testing instructions
- Verify the ELEC ECAM works as expected for regular and abnormal operations.
- Verify the battery push buttons work as expected.
- Verify the battery voltage display on the overhead panel works as expected.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
